tests_integration/test_connector_flows.py

```diff
@@ -176,12 +152,12 @@ def test_connector_flows_image_change(run_kopf, random_connector_name):
         # Change image tag
         # kubectl patch tc/test-connector-image-local -p '{"spec": {"image": {"tag": "1.63.0"}}}' --type=merge
         kubectl_patch(f"tc/{connector_name}", {"spec": {"image": {"tag": "1.63.0"}}})
-        time.sleep(10)
-        pod = kubectl_get("pod", connector_name)
+        time.sleep(5)
+        pod = kubectl_wait_pod_running(connector_name)
         assert pod["status"]["phase"] == "Running"
         assert pod["spec"]["containers"][0]["image"] == "twingate/connector:1.63.0"
 
-        kubectl_delete(f"tc/{connector_name}")
+        kubectl_delete_wait("tc", connector_name)
         time.sleep(10)
```

Comment:
We can delete this `sleep` now? 🤔

Reply:
No, we delete the TwinGateConnector object but it takes time for k8s to propogate the change to its child obejects and delete them too